russian-gost-r-705-2008-numeric-citations-alphabetical-bibliography.csl

```diff
@@ -0,0 +1,175 @@
+      <label prefix=" " suffix=" "/>
+      <name initialize-with="."/>
+    </names>
+  </macro>
+  <macro name="publisher">
+    <group suffix=",">
+      <text variable="publisher-place" suffix=":"/>
+      <text variable="publisher" prefix=" "/>
+    </group>
+    <text macro="year-date" prefix=" " suffix="."/>
+  </macro>
+  <macro name="year-date">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="language-sort">
+    <text variable="note"/>
+    <text value="Z999"/>
+  </macro>
+  <macro name="citation-number">
+    <text variable="citation-number" suffix=". "/>
+  </macro>
+  <macro name="author-first">
+    <names variable="author">
+      <name delimiter-precedes-last="always" et-al-min="4" et-al-use-first="1" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
+      <label form="short" strip-periods="true" prefix=" " suffix="."/>
+    </names>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout prefix="[" suffix="]" delimiter=", ">
+      <text variable="citation-number"/>
+      <group prefix=",">
+        <label plural="never" prefix=" " variable="locator" form="short"/>
+        <text variable="locator" form="short" prefix=" "/>
+      </group>
+    </layout>
+  </citation>
+  <locale xml:lang="en">
+    <terms>
+      <term name="et-al"> [и др.].</term>
+      <term name="editor">под ред.</term>
+      <term name="accessed">дата обращения</term>
+      <term name="page" form="short">с.</term>
+      <term name="translator">перевод</term>
+    </terms>
+  </locale>
+  <locale xml:lang="ru">
+    <terms>
+      <term name="et-al"> [и др.].</term>
+      <term name="editor">под ред.</term>
+      <term name="accessed">дата обращения</term>
+      <term name="page" form="short">с.</term>
+      <term name="translator">перевод</term>
+    </terms>
+  </locale>
+  <bibliography>
+    <sort>
+      <key macro="language-sort"/>
+      <key macro="author-first"/>
+      <key variable="issued"/>
+    </sort>
+    <layout suffix=".">
+      <text macro="citation-number"/>
+      <text macro="author-first" suffix=" "/>
+      <choose>
+        <if type="bill book graphic legal_case motion_picture report song" match="any">
+          <group delimiter="" prefix="">
+            <group delimiter=" ">
+              <text macro="title"/>
+              <text macro="author"/>
+              <text macro="editor" suffix=" "/>
+              <text macro="translator"/>
+              <text variable="edition" prefix=" " suffix="-е изд.,"/>
+            </group>
+            <text macro="publisher" prefix=" "/>
+            <text variable="number-of-pages" prefix=" " suffix=" c."/>
+          </group>
+        </if>
+        <else-if type="chapter paper-conference" match="any">
+          <group prefix=" " delimiter="">
```

Comment:
delete the prefix (you have a space after the author above with a suffix) and the empty delimiter.